src/commands/color.hpp

```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include <all.hpp>
+
+class ColorCommand : public liteshell::BaseCommand
+{
+public:
+    ColorCommand()
+        : liteshell::BaseCommand(
+              "color",
+              "Change the text color in the shell",
+              "Specify a color in hex format to change the text color",
+              liteshell::CommandConstraint("color", "The color to set", true, false)) {}
+
+    DWORD run(const liteshell::Context &context)
+    {
+        std::string color = context.get("color");
+        if (!utils::setColor(color))
+        {
+            throw std::invalid_argument("Error: Unable to set color \"" + color + "\"");
+        }
```

Comment:
`utils::setColor(color)` returning value does not matter anymore.
```suggestion
        utils::setColor(color);
```